phpunit/class-block-templates-test.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * REST API: Block_Templates_Test class
+ *
+ * @package    WordPress
+ * @subpackage REST_API
+ */
+
+/**
+ * Tests for the Block Templates abstraction layer.
+ */
+class Block_Templates_Test extends WP_UnitTestCase {
+	private static $post;
+
+	public static function wpSetUpBeforeClass() {
+		switch_theme( 'tt1-blocks' );
+		gutenberg_register_template_post_type();
+		gutenberg_register_template_part_post_type();
+		gutenberg_register_wp_theme_taxonomy();
+
+		$args = array(
+			'post_type'    => 'wp_template',
+			'post_name'    => 'my_template',
+			'post_title'   => 'My Template',
+			'post_content' => 'Content',
+			'post_excerpt' => 'Description of my template',
+			'tax_input'    => array(
+				'wp_theme' => array(
+					get_stylesheet(),
+				),
+			),
+		);
+
+		self::assertTrue( taxonomy_exists( 'wp_theme' ) );
```

Comment:
Please no assertions in the setup functions. Add it to it's own test if you need to make sure it exists.

Reply:
Yeah this was just here for testing purposes to ensure the taxonomy is being registered, it can be dropped.

Reply:
Removed in 3c246e360e.